.github/workflows/ci.yaml

```diff
@@ -0,0 +1,47 @@
+name: CI-CD
+on: [push]
+jobs:
+  test_push:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+
+      - name: npm install
+        run: npm i
+
+      - name: Run unit tests
+        run: npm test
+
+  release:
+    name: Release
+    # https://github.community/t/trigger-job-on-tag-push-only/18076
+    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
```

Comment:
Why not use:
```yaml
on:
  push:
    branches:
      - master
```
I assume you're trying to limit this to pushes to the master branch correct?

Reply:
This is just a preparation for extended push-pull flow. Anyway, fixed for now.